codex-cli/src/utils/agent/handle-exec-command.ts

```diff
@@ -213,15 +216,24 @@ async function execCommand(
   runInSandbox: boolean,
   additionalWritableRoots: ReadonlyArray<string>,
   abortSignal?: AbortSignal,
+  config?: AppConfig,
```

Comment:
Can we make this required? (Not sure why `abortSignal` is optional: I'll have to investigate.) At 6 params, we should really be taking a single dict rather than individual args, but we can do that in a separate PR.